python/src/opendp/extras/sklearn/decomposition/__init__.py

```diff
@@ -180,139 +180,191 @@ def __init__(
         whiten: bool = False,
     ) -> None:
```

Comment:
```suggestion
    ) -> None:  # pragma: no cover
```
Would this work, instead of putting it on every line inside?